### DISCUSSION TITLE
Apps Volume Control

### DISCUSSION THREAD

**Discussion by Tiag0Carvalh0** [NicoHood/HID#363] [Q&A]
Hello there,
For my project i want to controll spotify volume with a encoder, i can see that the command MEDIA_VOLUME_UP/DOWN, works, but it lowers all system volume, and i want just 1 specific app(spotify), i am using toastify, to bind new keys, but i cant find a command to bind volume up and donw, just the MEDIA_VOLUME_UP, if i bind it, i think i "bug" windows, like it does nothing.
Can someone help please?

Whatching commands from https://github.com/NicoHood/HID/blob/master/src/HID-APIs/ConsumerAPI.h

**Comment by Tiag0Carvalh0** [comment]
Just Used Keyboard  with  BootKeyboard.write(  KEY_F23  ); xD
But now i have a class with the same name, of another lib look 
![Sem Título](https://user-images.githubusercontent.com/63639112/173553599-3431c0b5-8157-45f5-b3d0-85c6c82322a7.png)

**Comment by bnnbgs** [comment]
How did you solved it ? i have the same problem, i want to control only one specific program and i can´t configurate it

**Reply by Tiag0Carvalh0** [reply]
i am using deej at the moment.
